pyproject.toml

```diff
@@ -71,33 +71,35 @@ all = [
     "astropy[ipython]",
     "astropy[jupyter]",
     # Install all remaining optional dependencies
-    "certifi",
-    "dask[array]>=2022.8.1",
-    "h5py",
-    "pyarrow>=10.0.1",
-    "beautifulsoup4",
-    "html5lib",
-    "bleach",
-    "pandas>=2.0",
-    "sortedcontainers",
-    "pytz",
-    "jplephem",
-    "mpmath",
-    "asdf-astropy>=0.3",
-    "bottleneck",
+    "certifi>=2022.6.15.1",
+    "dask[array]>=2022.5.1",
```

Comment:
For dask I was able to actually test this lower bound (the previous one, `2022.8.1`, was just a guess I took in #16903)